templates/zerver/api/get-messages.md

```diff
@@ -52,68 +52,7 @@ When a request is successful, this endpoint returns a dictionary
 containing the following (in addition to the `msg` and `result` keys
 present in all Zulip API responses).
 
-* `anchor`: the same `anchor` specified in the request.
-* `found_newest`: whether the `messages` list includes the latest message in
-    the narrow.
-* `found_oldest`: whether the `messages` list includes the oldest message in
-    the narrow.
-* `found_anchor`: whether it was possible to fetch the requested anchor, or
-    the closest in the narrow has been used.
-* `messages`: an array of `message` objects, each containing the following
-    fields:
-    * `avatar_url`: The URL of the user's avatar.
-    * `client`: A Zulip "client" string, describing what Zulip client
-      sent the message.
-    * `content`: The content/body of the message.
-    * `content_type`: The HTTP `content_type` for the message content.  This
-      will be `text/html` or `text/x-markdown`, depending on
-      whether `apply_markdown` was set.
-    * `display_recipient`: Data on the recipient of the message;
-      either the name of a stream or a dictionary containing data on
-      the users who received the message.
-    * `flags`: The user's [message flags][message-flags] for the message.
-    * `id`: The unique message ID.  Messages should always be
-      displayed sorted by ID.
-    * `is_me_message`: Whether the message is a [/me status message][status-messages]
-    * `reactions`: Data on any reactions to the message.
-        * `emoji_code`: An encoded version of the emoji's unicode codepoint.
-        * `emoji_name`: Name of the emoji.
-        * `reaction_type`: If the reaction uses a [custom emoji](/help/add-custom-emoji),
-          `reaction_type` will be set to `realm_emoji`.
-        * `user_id`: The ID of the user who added the reaction.
-           **Changes**: New in Zulip 2.2 (feature level 2). The `user`
-           object is deprecated and will be removed in the future.
```

Comment:
In the zulip.yaml, can you put a blank line before the `Changes` entry so it stands out better?

same for deprecated below.

Reply:
Done for all changes.